packages/foam-vscode/src/test/web/web-extension.webtest.ts

```diff
@@ -0,0 +1,11 @@
+/*global suite:readonly*/
+
+import * as assert from 'assert';
+import * as vscode from 'vscode';
+suite('Foam Web Extension Test Suite', () => {
+  vscode.window.showInformationMessage('Start all tests.');
+
+  test('Foam extension is loaded', () => {
+    assert.ok(vscode.extensions.getExtension('foam.foam-vscode'));
```

Comment:
I like the idea of doing just a quick smoke check here, making sure that:
- the extension is present
- the API is exposed
- a few commands are available

I think that should be enough to get started

Reply:
I am stuck at expanding the tests. See https://github.com/microsoft/vscode-test-web/issues/130.